src/app/shared/team-radio-switch/team-radio-switch.component.html

```diff
@@ -8,8 +8,7 @@
       [(ngModel)]="valueId"
     />
     <app-team-logo
-      [name]="team1.name"
-      [iconUrl]="team1.iconUrl"
+      [teamId]="111"
```

Comment:
RefTeam.Id ?

Reply:
In this way i am take from input checked property

Reply:
fixed